### PR TITLE
Add common.properties file at root level of datadir

### PR DESCRIPTION
**Pull request by Vampouille**
This file will provide "default" values for all webapps.
Values can be overrided by webapps.

### CODE REVIEW

File: commons/src/main/java/org/georchestra/commons/configuration/GeorchestraConfiguration.java [hunk 1]
```diff
@@ -66,22 +67,15 @@ public GeorchestraConfiguration(String context) {
...
-                activated = false;
-                return;
-            }
+                this.applicationSpecificProperties = this.loadCustomPropertiesFile(context);
+            } catch (Exception e) {}
+
+            // loads common context property file
+            try {
+                this.commonProperties = this.loadPropertiesFile(new File(String.format("%s%s%s",
+                        globalDatadir, File.separator, "common.properties")));
```
Comment (fvanderbiest) [inline commons/src/main/java/org/georchestra/commons/configuration/GeorchestraConfiguration.java]: shared.properties ?

Comment (landryb) [reply commons/src/main/java/org/georchestra/commons/configuration/GeorchestraConfiguration.java]: wont this confuse/bring bad memories about shared.properties in the template build ? :)

Comment (fvanderbiest) [reply commons/src/main/java/org/georchestra/commons/configuration/GeorchestraConfiguration.java]: You're right, but we want to get rid of the config jar ...

Comment (fvanderbiest) [reply commons/src/main/java/org/georchestra/commons/configuration/GeorchestraConfiguration.java]: In addition, you're free not to use this shared.properties file

Comment (Vampouille) [reply commons/src/main/java/org/georchestra/commons/configuration/GeorchestraConfiguration.java]: What about `default.properties` ? This name explicitly explain priority between webapp properties and common/shared/default one

Comment (Vampouille) [reply commons/src/main/java/org/georchestra/commons/configuration/GeorchestraConfiguration.java]: > What about default.properties ?

Done with 0ebecfd

**Pull request merged**